portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/dataset/DataSetViewFilters.testcase

```diff
@@ -238,6 +238,242 @@ definition {
+	@priority = 5
+	test AssertTheTranslationInFilters {
+		task ("And creating a picklist") {
+			Picklist.addPicklistViaAPI(picklistName = "Transport");
+		}
+
+		task ("And Add items to the created picklist") {
+			Picklist.addPicklistItemViaAPI(
+				itemKey = "bike",
+				itemName = "bike",
+				picklistName = "Transport");
+
+			Picklist.addPicklistItemViaAPI(
+				itemKey = "car",
+				itemName = "car",
+				picklistName = "Transport");
+
+			Picklist.addPicklistItemViaAPI(
+				itemKey = "bus",
+				itemName = "bus",
+				picklistName = "Transport");
+		}
+
+		task ("When the user accesses the Fields tab") {
+			DataSetAdmin.goToTab(tabName = "Fields");
+		}
+
+		task ("And adds a new filter") {
+			LexiconEntry.gotoAdd();
+		}
+
+		task ("And Check two options (dateCreated,dateModified)") {
+			DataSetAdmin.checkFields(key_fieldList = "dateCreated,dateModified");
+
+			Button.clickSave();
+		}
+
+		task ("When the user accesses the Filters tab") {
+			DataSetAdmin.goToTab(tabName = "Filters");
+		}
+
+		task ("And Choose a Selection option") {
+			LexiconEntry.gotoAdd();
+
+			Click(
+				key_fieldName = "Selection",
+				locator1 = "FormViewBuilder#OBJECT_FIELD_BUTTON_DROPDOWN");
+
+			Click(
+				key_symbolName = "",
+				locator1 = "FormViewBuilder#BUTTON_LOCALIZATION");
+
+			Click(
+				locator1 = "CommerceWidget#TAG_FACET_COMPACT_LAYOUT",
+				tagName = "pt_BR");
+
+			Click(
+				locator1 = "Portlet#MODAL_TITLE",
+				value1 = "New Selection Filter");
+
+			Type(
+				key_text = "Name",
+				locator1 = "TextInput#ANY",
+				value1 = "PT-BR");
+
+			Click(
+				key_class = "btn-secondary",
+				key_index = 3,
+				locator1 = "Button#ANY_WITH_CLASS_AND_INDEX");
+
+			SelectFrameTop();
+
+			Click.javaScriptClick(
+				key_sortOption = "Relationship_c_fdsViewERC",
+				locator1 = "ObjectCustomViews#SORT_OPTION");
+		}
+
+		task ("And selects the Transport option in the Source Option drop-down") {
+			Select(
+				locator1 = "DataSet#SOURCE_OPTIONS_SELECT",
+				value1 = "Transport");
+		}
+
+		task ("And Preselected Values car,bike,bus ") {
+			Click(
+				key_fieldName = "Select a default value for your filter.",
+				locator1 = "ObjectCustomViews#FROM_DATE_INPUT");
+
+			Click(
+				key_sortOption = "car",
+				locator1 = "ObjectCustomViews#SORT_OPTION");
+
+			Click(
+				key_sortOption = "bike",
+				locator1 = "ObjectCustomViews#SORT_OPTION");
+
+			Click(
+				key_sortOption = "bus",
+				locator1 = "ObjectCustomViews#SORT_OPTION");
+		}
+
+		task ("And Select the option Multiple,Include radio button") {
+			Click.clickAtNotVisible(
+				key_radioOption = "Multiple",
+				locator1 = "CommerceEntry#RADIO_BUTTON");
+
+			Click.clickAtNotVisible(
+				key_radioOption = "Include",
+				locator1 = "CommerceEntry#RADIO_BUTTON");
+		}
+
+		task ("And the user save the change clicking on save button") {
+			Button.clickSave();
+		}
```

Comment:
I think all this can be done in a macro, passing all the data as parameters, so we would have a cleaner testm what do you think?

Reply:
Done!